version.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+# This script finds and prints the various versions in this project: wasi-sdk
+# itself, LLVM, and the Git revisions of dependencies.
+#
+# Usage: version [wasi-sdk|llvm|llvm-major|dump] [--llvm-dir=<non-project dir>]
+
+import sys
+import argparse
+import subprocess
+
+# The number of characters to use for the abbreviated Git revision.
+GIT_REF_LEN = 12
+
+
+def exec(command, cwd=None):
+    result = subprocess.run(
+        command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True, cwd=cwd)
+    if result.returncode != 0:
```

Comment:
might be simpler to just add `check=True`, but lgtm either way.

Reply:
And drop the `stderr=subprocess.PIPE`, right?

Reply:
Right that would be my recommendation for keeping this code simpler and letting python itself report the errors.